pymc3/variational/inference.py

```diff
@@ -618,3 +618,7 @@ def fit(n=10000, local_rv=None, method='advi', model=None, random_seed=None, sta
                         'or Inference instance' %
                         set(_select.keys()))
     return inference.fit(n, **kwargs)
+
+def _infmean(input_array):
```

Comment:
Suppose to use only locally in module. Maybe it's better to move inside local namespace